python/lsst/afw/math/_transformBoundedField.cc

```diff
@@ -37,33 +38,33 @@ using namespace py::literals;
 namespace lsst {
 namespace afw {
 namespace math {
-namespace {
 
 using ClsField = py::class_<TransformBoundedField, std::shared_ptr<TransformBoundedField>, BoundedField>;
 
-PYBIND11_MODULE(transformBoundedField, mod) {
-    /* Module level */
-    ClsField cls(mod, "TransformBoundedField");
+void wrapTransformBoundedField(lsst::utils::python::WrapperCollection &wrappers) {
+     wrappers.wrapType(ClsField (wrappers.module,"TransformBoundedField"),
```

Comment:
Needs `addSignatureDependency("lsst.afw.table.io")` to support `addPersistableMethods`.